group/group.go

```diff
@@ -11,9 +14,11 @@ type Group interface {
 type Element interface {
 	ScalarMultiplication(e Element, scalar fr.Element) Element
 	Set(e Element) Element
+	Add(a, b Element) Element
 	AddAssign(e Element) Element
 	Equal(e Element) bool
 	Bytes() []byte
+	MultiExp([]Element, []fr.Element) (Element, error)
```

Comment:
We need these new methods for group. 
`Add` is mostly a twist of `AddAssign`; you'll see in the impl.
`MultiExp` is a MSM.